generateSimLaunchScript.py

```diff
@@ -71,7 +72,7 @@
 for sampleNumber, outDir, simTitle in [(trainingSampleNumber, trainingOutDir, "training data"), (testSampleNumber, testOutDir, "test data")]:
     partialStr = "-Pc 0.2 0.99"
     print("\n#generating %s\n" %(simTitle))
-    neutDiscoalCmd = "discoal -i 4 %d %d %d -Pt %f %f -Pre %f %f%s" %(sampleSize, sampleNumber, numSites, thetaLow, thetaHigh, rhoMean, rhoMax, demogStr)
+    neutDiscoalCmd = "discoal %d %d %d -Pt %f %f -Pre %f %f%s -i 4" %(sampleSize, sampleNumber, numSites, thetaLow, thetaHigh, rhoMean, rhoMax, demogStr)
```

Comment:
can you leave the -i 4 out of the PR? I wouldn't want people running with that option by default

Reply:
sure, I kept it bc it was in the original